packages/dds/merge-tree/src/partialLengths.ts

```diff
@@ -454,25 +627,68 @@ export class PartialSequenceLengths {
 		combinedPartialLengths: PartialSequenceLengths,
 		segment: ISegment,
 		removalInfo?: IRemovalInfo,
+		moveInfo?: IMoveInfo,
```

Comment:
the doc comment on this function should be updated as well (probably just "If `removalInfo`, `moveInfo`, or both are defined, this operation updated the bookkeeping to account for the (re)moval of the segment instead")